test/mocks/event/mocks.h

```diff
@@ -214,6 +214,7 @@ class MockFileEvent : public FileEvent {
 
   MOCK_METHOD(void, activate, (uint32_t events));
   MOCK_METHOD(void, setEnabled, (uint32_t events));
+  MOCK_METHOD(uint32_t, getEnabled, ());
```

Comment:
Is this still needed?

Reply:
Good catch, no it is not needed. Let me remove it

Reply:
removed in 5153b1eb0359af6fd5a7b00eef747f16a6b71f26